### DISCUSSION TITLE
update problem running in docker container with fly.io

### DISCUSSION THREAD

**Discussion by lokize** [pocketbase/pocketbase#2826] [Q&A]
i have this problem when i try update with ./pocketbase update command from my docker container running in fly.io:

![image](https://github.com/pocketbase/pocketbase/assets/19701630/ad6378fd-6881-4905-9b53-8297b2d221dd)

**Comment by ganigeorgiev** [comment]
The `update` command doesn't make much sense in a Docker environment because with the next image rebuild you'll lose your changes (_there is a task to add info message when running the command inside docker_).

If you want to update the PocketBae version in a fly.io configuration, the correct approach would be to update either the Dockerfile `PB_VERSION` env OR modify it from your fly.tom by using the `build.args` section:

```toml
[build.args]
  PB_VERSION="0.16.6"
```

**Reply by lokize** [reply]
oh nevermind, really, I just updated my dockerfile with the new version and deployed it to fly.io, thanks!
